internal/report/store_memory_test.go

```diff
@@ -0,0 +1,173 @@
+package report_test
+
+import (
+	"github.com/stretchr/testify/require"
```

Comment:
I would prefer not to add a (dev) dependency if it isn't needed, maybe we can do something else besides `require.NoError()`.

Reply:
make sense, I ended up using a plain nil-check for errors and removed `require` as a dependency

Reply:
Many thanks!